src/features/timebank/timebank-API-provider.ts

```diff
@@ -13,13 +14,14 @@ namespace TimeBankApiProvider {
    */
   export const getTimebankUsers = async (): Promise<PersonDto[]> => {
     try {
-      const client = new TimebankApi(process.env.timebank_base_url);
       const { body } = await client.timebankControllerGetPersons();
-
+      if(body.length === 0){
```

Comment:
You can check for 0 length or undefined with !body.length
```suggestion
      if (!body.length) {
```